src/js-transform.ts

```diff
@@ -156,7 +161,19 @@ export function jsTransform(js: string, options: JsTransformOptions): string {
   }
 
   if (doBabel) {
-    js = babelCore.transform(js, {presets, plugins}).code!;
+    try {
+      js = babelCore.transform(js, {presets, plugins}).code!;
+    } catch (e) {
```

Comment:
This will catch more than just parse errors. Maybe we should inspect the errors?

Reply:
Done. Is this too brittle? Also renamed to `softSyntaxError` to match Babel's terminology.

Reply:
What do you think about the (roughly equivalent): `e.constructor.name === 'SyntaxError'`